systemtest/tests/integration/test.sh

```diff
@@ -42,7 +42,6 @@ EOF
 ./integration-tests/scripts/run-local-candlepin.sh
 
 # create testing data in local candlepin
-./integration-tests/scripts/post-activation-keys.sh
```

Comment:
This change should not be part of this PR. It is already part of another PR.

Reply:
ok, I will revert this change .

Reply:
reverted